drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c

```diff
@@ -872,6 +872,38 @@ unsigned int tx_buff_req_free(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 				/*Adjust the counters*/
 				fmac_dev_ctx->tx_config.outstanding_descs[tx_done_q]--;
 				fmac_dev_ctx->tx_config.outstanding_descs[*ac]++;
+
+				/*Update the queue_map*/
+				switch (desc %
+					(fpriv->num_tx_tokens_per_ac * WIFI_NRF_FMAC_AC_MAX)) {
```

Comment:
Maybe store the value of (desc % (fpriv->num_tx_tokens_per_ac * WIFI_NRF_FMAC_AC_MAX)) in a variable?